.travis.yml

```diff
@@ -22,11 +22,17 @@ matrix:
         - '! find . -type f -name "*.php" -exec php -d error_reporting=32767 -l {} \; 2>&1 >&- | grep "^"'
       script: phpunit --bootstrap expandFns.php tests/phpunit
     - stage: deploy
-      name: "Update WMFlabs servers"
+      name: "Update WMFlabs servers and test them"
       php: 5.6 # Avoid downloading and installing 5.5
       script:
         - if [ "$TRAVIS_BRANCH" = "master"      -a "$TRAVIS_PULL_REQUEST" = "false" ]; then wget -O- https://tools.wmflabs.org/citations/gitpull.php ; fi
         - if [ "$TRAVIS_BRANCH" = "development" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then wget -O- https://tools.wmflabs.org/citations-dev/gitpull.php ; fi
+        - if [ "$TRAVIS_BRANCH" = "master"      -a "$TRAVIS_PULL_REQUEST" = "false" ]; then curl "https://tools.wmflabs.org/citations/category.php?cat=X5" ; fi
+        - if [ "$TRAVIS_BRANCH" = "development" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then curl "https://tools.wmflabs.org/citations-dev/category.php?cat=X5" ; fi
```

Comment:
Won't this test the code that's on the server already?  The changes being proposed will not be pulled onto tool labs until they've been merged.